service/controller/key/ami.go

```diff
@@ -0,0 +1,176 @@
+// NOTE: This file is generated. Do not edit.
```

Comment:
We need docs for that. 

* Where does it come from? 
* How is it generated? 
* Who takes care of this? 
* When do we need to update this?

Reply:
I have addressed this in `README.md`, and this comment will be changed to mention that it was generated by `devctl` momentarily. Let me know if this is sufficient.